pkg/scaffold/project/kustomize.go

```diff
@@ -65,21 +65,9 @@ namePrefix: {{.Prefix}}-
 #commonLabels:
 #  someName: someValue
 
-# Each entry in this list must resolve to an existing
-# resource definition in YAML.  These are the resource
-# files that kustomize reads, modifies and emits as a
-# YAML string, with resources separated by document
-# markers ("---").
-resources:
-- ../rbac/rbac_role.yaml
-- ../rbac/rbac_role_binding.yaml
-- ../manager/manager.yaml
-  # Comment the following 3 lines if you want to disable
-  # the auth proxy (https://github.com/brancz/kube-rbac-proxy)
-  # which protects your /metrics endpoint.
-- ../rbac/auth_proxy_service.yaml
-- ../rbac/auth_proxy_role.yaml
-- ../rbac/auth_proxy_role_binding.yaml
+bases:
+- ../rbac
```

Comment:
I have to double check, but rbac manifests get generated everytime make runs, so not sure if delete all yamls in `rbac` directory.

Reply:
IIRC it doesn't delete everything, it overwrites the content in `../rbac/rbac_role.yaml`. We'd better confirm it.

Reply:
I confirmed, it doesn't delete, so the change is good.